src/lib/token.js

```diff
@@ -6,10 +6,4 @@ export const generateToken = (data, secret, expiresInSeconds) => jwt.sign(
     { expiresIn: `${expiresInSeconds} seconds`, algorithm: "HS256" }
 );
 
-export const verifyAndDecodeToken = (token, secret) => {
-    try {
-        return jwt.verify(token, secret, { algorithm: "HS256" });
-    } catch (err) {
-        return null;
-    }
-};
+export const verifyAndDecodeToken = (token, secret) => jwt.verify(token, secret, { algorithm: "HS256" });
```

Comment:
Was this updated in the recover password routes?

Reply:
yes